.github/workflows/src/label-rules.js

```diff
@@ -0,0 +1,864 @@
+/*
+    This file determines what labels are required for a given PR. It uses this context to obtain two key
+    pieces of information:
+    1. The labels that are currently present on the PR.
+    2. The target branch of the PR.
+*/
+
+import {
+  brchTsg,
+  diagramTsg,
+  href,
+  notReadyForArmReviewReason,
+  wrapInArmReviewMessage,
+} from "./tsgs.js";
+
+/**
+ * This file is the single source of truth for the labels used by the SDK generation tooling
+ * in the Azure/azure-rest-api-specs and Azure/azure-rest-api-specs-pr repositories.
+ *
+ * For additional context, see:
+ * - https://gist.github.com/raych1/353949d19371b69fb82a10dd70032a51
+ * - https://github.com/Azure/azure-sdk-tools/issues/6327
+ * - https://microsoftapc-my.sharepoint.com/:w:/g/personal/raychen_microsoft_com/EbOAA9SkhQhGlgxtf7mc0kUB-25bFue0EFbXKXS3TFLTQA
+ */
+
+export const sdkLabels = {
+  "azure-cli-extensions": {
+    breakingChange: undefined,
```

Comment:
These breaking change labels are also present at `eng/tools/openapi-diff-runner/src/types/breaking-change.ts`. That's a compiled solution though. I'm betting I need to reverse the locoations, so breaking-change.ts refers to an external file in `.github/shared` to get these breaking change label names.